package.json

```diff
@@ -33,7 +33,7 @@
     "benchmark": "node benchmark/bench.js",
     "install": "node scripts/install.js",
     "prepare": "tsc -p lib && tsc -p test && tsc -p benchmark",
-    "test": "mocha test --recursive",
+    "test": "mocha test -g \"^((?!napajs/timers).)*$\" --recursive && mocha test -g \"^napajs/timers\"",
```

Comment:
What does this part do

Reply:
other test affect result of the schedule precision. So split timer related.